src/fs-poll.c

```diff
@@ -104,19 +116,17 @@ int uv_fs_poll_start(uv_fs_poll_t* handle,
 int uv_fs_poll_stop(uv_fs_poll_t* handle) {
   struct poll_ctx* ctx;
 
-  if (!uv__is_active(handle))
+  if (!uv_is_active((uv_handle_t*)handle))
```

Comment:
I wonder if this could lead to divergent behavior on Windows and Unices since the former's uv_is_active() also checks that the UV_HANDLE_CLOSING flag is unset.

(That's probably something we should fix regardless of this PR.)

Reply:
Looking at it in detail, I think it’s okay here, because closing the `uv_fs_poll_t` also stops it. But yes, that seems like an inconsistency that should be fixed.